pkg/detectors/detectors.go

```diff
@@ -92,7 +92,7 @@ func CleanResults(results []Result) []Result {
 func PrefixRegex(keywords []string) string {
 	pre := `(?i)(?:`
 	middle := strings.Join(keywords, "|")
-	post := `).{0,40}`
+	post := `).[\r\n]{0,40}`
```

Comment:
This needs to be `[.\r\n]` otherwise it captures one character and up to 40 CR/LF

Reply:
Had to separate out the class group from the `.` and put them all into a capture group. That seems to have worked properly.